src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -753,6 +802,18 @@ def create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
         utils.arm_exception_handler(e, consts.Create_ConnectedCluster_Fault_Type, 'Unable to create connected cluster resource')
 
 
+def patch_cc_resource(client, resource_group_name, cluster_name, cc, no_wait, enable_private_link):
+    try:
+        if enable_private_link:
+            return sdk_no_wait(no_wait, client.begin_update, resource_group_name=resource_group_name,
```

Comment:
why don't we wait for this operation to complete similar to PUT? What is error experience for patch failure scenarios?

Reply:
resolving as we wait for patch async operation first